libtsuba/include/tsuba/RDGPrefix.h

```diff
@@ -16,7 +16,8 @@ class RDGManifest;
 /// partitioning decisions
 class KATANA_EXPORT RDGPrefix {
 public:
-  static katana::Result<RDGPrefix> Make(RDGHandle handle);
+  static katana::Result<RDGPrefix> Make(
```

Comment:
nit: I would prefer to avoid defaulting `partition_id` if reasonably possible.
Are there a ton of use cases where the caller of `Make` doesn't have any way of knowing what `partition_id` it should use?

Reply:
The largest benefit of this is that it makes the interface change non-breaking. I would be happy to come back and remove the default after all uses of this interface have been updated.